packages/ERTP/test/unitTests/test-issuerObj.js

```diff
@@ -101,28 +101,35 @@ test('issuer.makeEmptyPurse', t => {
     .catch(e => t.assert(false, e));
 });
 
-test('issuer.deposit', t => {
-  t.plan(2);
```

Comment:
Why drop `t.plan()`? I realize that in the general case it's more work to maintain a number that might change. In this case, the test isn't as likely to require maintenance as the code is. 

I think if the deposit code breaks in a way that causes the the result to be a broken promise, this test will silently succeed, because the `.then()` clauses don't have `onRejected()` clauses. Hopefully, we'd have other tests that would catch that, but this test should also detect failure in that circumstance, I think.

``` js
await E(purse)
    .deposit(payment17, fungible17)
    .then(checkDeposit(fungible0, fungible17));
  await E(purse)
    .deposit(payment25, fungible25)
    .then(checkDeposit(fungible17, fungibleSum));
});	
```

Reply:
The `await`s mean that a missing `.then()` or `.catch()` will result in the test returning a rejected promise, and a "test exited without ending" error message.  It needs a `try...catch...finally` around the whole test in order to be less messy when it fails, but I don't see the reason for a `t.plan`.

Reply:
Actually, in this test style, the test framework waits for the promise, and if it's breaks, the test fails. So this is actually quite nice. The argument for `plan` is belt and suspenders for making sure that the `asserts` in the abstraction actually get tried. Occasionally people try to do clever tests with async stuff such that some asserts never get tested

Reply:
I think what Michael is saying is that the test failure cascades for every subsequent test, unless you catch it.